encoding/json/outbound.go

```diff
@@ -52,6 +59,9 @@ type Request struct {
 	// Request body. This may be any type that can be serialized by
 	// json.Marshal.
 	Body interface{}
+
+	// Ttlms is the ttl in ms
+	Ttlms int
```

Comment:
will make issue for this - we should talk with @kriskowal about what we want this variable to be called in all langs, and what it should look like on the wire for HTTP